netlify.toml

```diff
@@ -0,0 +1,6 @@
+# https://docs.netlify.com/configure-builds/file-based-configuration/
```

Comment:
I'm encouraging us to use the toml for non-secrets because it makes the configuration really clear to non-admins

Reply:
Nice! I'm all for it 🙌